src/Persoane/Client.java

```diff
@@ -0,0 +1,89 @@
+package Persoane;
+
+
+import Comanda.Comanda;
+
+import java.util.ArrayList;
+import java.util.Objects;
+
+public class Client extends Persoana {
+
+    private int id;
+    private String adresa;
+    private Comanda comanda;
+    private ArrayList<Comanda> comenzi;
```

Comment:
List<Comanda> instead of ArrayList<Comanda> is best practice to use the abstractization for the reference